lib/rubocop/cop/security/json_load.rb

```diff
@@ -3,7 +3,8 @@
 module RuboCop
   module Cop
     module Security
-      # This cop checks for the use of unsecure JSON methods.
+      # This cop checks for the use of JSON class methods which have potential
+      # security issues.
```

Comment:
I'd extend this class comment with explanation why autocorrect can't be implemented reliably for this.